gridt/src/app/login/register/register.page.ts

```diff
@@ -66,4 +68,12 @@ export class RegisterPage implements OnInit {
       .then(alertEl => alertEl.present());
   }
 
+  async presentPrivacy() {
+    const modal = await this.modalController.create({
+      component: PrivacyPage
+    });
+    console.log("present privacy policy");
```

Comment:
In general we try to keep the standard log level as empty as possible to prevent cluttering. These type of messages normally get level "debug".

I would recommend a change to:
`console.debug("Present privacy policy");`